pkg/asset/installconfig/gcp/validation_test.go

```diff
@@ -122,6 +127,12 @@ func TestGCPInstallConfigValidation(t *testing.T) {
 			expectedError:  true,
 			expectedErrMsg: "computeSubnet: Invalid value.*controlPlaneSubnet: Invalid value",
 		},
+		{
+			name:           "Invalid both machine types",
```

Comment:
Shouldn't this test case need an expected call to `GetMachineTypes` added to the GCP client mock?

Reply:
Yes, I forgot to generate the mock. https://github.com/openshift/installer/pull/4329#event-3949892939